test/powershell/engine/HelpSystem.OnlineHelp.Tests.ps1

```diff
@@ -0,0 +1,68 @@
+Describe 'Online help tests for PowerShell Core Cmdlets' -Tags "CI" {
+
+    $skipTests = [System.Management.Automation.Platform]::IsIoT -or [System.Management.Automation.Platform]::IsNanoServer -or [System.Management.Automation.Platform]::IsLinux
+
+    BeforeAll {
+
+        if (-not $skipTests)
+        {
+            # Verify that test resouces are available
+            $testFiles = @("$PSScriptRoot\testdata\HelpURI\V2Cmdlets.csv", "$PSScriptRoot\testdata\HelpURI\V3Cmdlets.csv")
+
+            # Enable the test hook
+            [system.management.automation.internal.internaltesthooks]::SetTestHook('BypassOnlineHelpRetrieval', $true)
+
+            # Remove the help content
+            $path = Join-Path $PSHOME (Get-UICulture).Name
+            Get-ChildItem "$path\*dll-help.xml" -Recurse | Remove-Item -Force -ErrorAction SilentlyContinue
+        }
+    }
+
+    AfterAll {
+        
+        if (-not $skipTests)
+        {
+            # Disable the test hook
+            [system.management.automation.internal.internaltesthooks]::SetTestHook('BypassOnlineHelpRetrieval', $false)
+        }
+    }
+
+    $testFiles | foreach {
+        $cmdletList = Import-Csv $_ -ea Stop
+
+        foreach ($cmdlet in $cmdletList)
+        {
+            # If the cmdlet is not preset in CoreCLR, skip it.
+            if (Get-Command $cmdlet.TopicTitle -ea SilentlyContinue)
+            {
+                # TopicTitle - is the cmdlet name in the csv file
+                # HelpURI - is the expected help URI in the csv file
+
+                It "Validate 'get-help $($cmdlet.TopicTitle) -Online'" -skip:($skipTests) {
+                    $actualURI = Get-Help $cmdlet.TopicTitle -Online
+                    $actualURI = $actualURI.Replace("Help URI:","").Trim()
```

Comment:
Thanks @JamesWTruher. I like the current implementation of the testhook, because it tells you that the fwlink is pointing to a Help URI.